presto-main/src/main/java/com/facebook/presto/operator/scalar/KleinbergBurstDetectionFunction.java

```diff
@@ -0,0 +1,200 @@
+package com.facebook.presto.operator.scalar;
```

Comment:
Please add the header for the newly added files, check: https://github.com/prestodb/presto#developers

Reply:
Done.